src/content/docs/distribute/Signing/ios.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: iOS Code Signing
+sidebar:
+  label: iOS
+  order: 4
+---
+
+Code signing on iOS requires enrolling to the [Apple Developer] program.
+
+To distribute iOS applications you must have your bundle identifier registered in the App Store Connect,
+an appropriate iOS code signing certificate and a mobile provisioning profile that links them together and enables the iOS capabilities used by your app.
+These requirements can be either automatically managed by Xcode or provided manually.
+
+### Automatic Signing
+
+Letting Xcode manage the signing and provisioning for your app is the most convenient way to export your iOS app to be distributed.
+It automatically registers your bundle identifier, manages iOS capabilities changes, and configures an appropriate certificate based on your export method.
+
+Automatic signing is enabled by default, and uses the account configured in Xcode to authenticate when used on your local machine.\
+To register your account, open the Xcode application and open the Settings page in the `Xcode > Settings` menu, switch to the Accounts tab and click the `+` icon.
+
+To use the automatic signing in CI/CD platforms you must create an App Store Connect API key
+and define the `APPLE_API_ISSUER`, `APPLE_API_KEY` and `APPLE_API_KEY_PATH` environment variables.\
+Open the [App Store Connect's Users and Access page], select the Keys tab, click on the Add button and select a name and the Developer access.
+The APPLE_API_ISSUER (Issuer ID) is presented above the keys table, and the APPLE_API_KEY is the value on the Key ID column on that table.
+You also need to download the private key, which can only be done once and is only visible after a page reload (the button is shown on the table row for the newly created key).
+The private key file path must be set via the APPLE_API_KEY_PATH environment variable.
+
+### Manual Signing
+
+To manually sign your iOS app you can provide the certificate and mobile provisioning profile via environment variables.
```

Comment:
We should explicitly explain and add the environment variables here. They are a little bit different than in the above automated signing section.